templates/security/email/reset_instructions.html

```diff
@@ -12,4 +12,4 @@
 
     <p><a href="{{ reset_link }}">{{ _fsdomain('Click here to reset your password') }}</a></p>
 
-{% endblock %}
\ No newline at end of file
+{% endblock %}
```

Comment:
this template should not be changed - it has nothing to do with reporting

Reply:
looks like my merging may have deleted this. or before the merge it was deleted from the master?

Reply:
ahhh no. i had my own version of reset_instruction.html and in the master it was rewritten to be shorter than my html thats why it was changed.

Reply:
remove the irrelevant change from your PR

Reply:
the entire file was overwritten, its not a line change.

Reply:
then make another commit to revert that one line change

Reply:
fine I will do it myself: https://github.com/zap-me/beryllium/pull/1/commits/f6a81d0f895d2110878840cc64011e2faadbec21